### PR TITLE
Refactor Code + ZPL license addition

### PR DESCRIPTION
**Pull request by yzainee**
1. Refactor code to remove the graph source and synonyms files from test to src

Added license 
ZPL 2.0: https://opensource.org/licenses/ZPL-2.0
https://tldrlegal.com/license/zope-public-license-2.0-(zpl-2.0)

ZPL 2.1: https://spdx.org/licenses/ZPL-2.1.html
https://tldrlegal.com/license/zope-public-license-2.1-(zpl-2.1)

More info:
https://en.wikipedia.org/wiki/Zope_Public_License

### CODE REVIEW

File: src/stack_license.py [hunk 1]
```diff
@@ -26,6 +25,7 @@
                               "GNU Affero General Public License", "The Apache License",
                               "The GNU Lesser General Public License", "version 3",
                               "The GNU General Public License", "version 2", "version 2.1"]
+LIC_DATA_DIR = os.environ.get("LIC_DATA_DIR", "src")
```
Comment (arajkumar) [inline src/stack_license.py:28]: This should be moved to src/config.py.

File: run-test-local.sh [hunk 2]
```diff
@@ -44,6 +44,7 @@ done
 
 export MAJORITY_THRESHOLD=$threshold
 export DATA_DIR=.
```
Comment (arajkumar) [inline run-test-local.sh:46]: I see that the usage of DATA_DIR has been removed, can you remove from here, runtests.sh and template.yaml?

**Review by arajkumar — COMMENTED** [review]
Changes looks good to me with few nits.

File: src/stack_license.py (hunk 1 above)
Comment (yzainee) [reply src/stack_license.py:28]: @arajkumar i dont have access to change that file. :) I tried and then gave up.

File: run-test-local.sh (hunk 2 above)
Comment (yzainee) [reply run-test-local.sh:46]: DATA_DIR is still used. actually to create the data set for running test cases. testdir1 - testdir4 gets created and are used in several tests.

File: src/stack_license.py (hunk 1 above)
Comment (yzainee) [reply src/stack_license.py]: done

File: run-test-local.sh (hunk 2 above)
Comment (arajkumar) [reply run-test-local.sh:46]: @yzainee , if it is used only in tests, can you remove from template.yaml?

Comment (yzainee) [reply run-test-local.sh:46]: done

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**